source/extensions/filters/network/tcp_proxy/config.h

```diff
@@ -8,7 +8,7 @@
 namespace Envoy {
 namespace Extensions {
 namespace NetworkFilters {
-namespace TcpProxy {
+namespace TcpProxyFilter {
```

Comment:
Why this change? I don't think any of the other network filters are named like this.

Reply:
It was because of namespace conflicts. With the other changes I can change it back.